frontend/src/App.js

```diff
@@ -109,19 +109,24 @@ function App() {
 									<Route exact path="/resetPassword">
 										<ResetPasswordPage setUser={setUser} user={user} />
 									</Route>
-									<UserArea user={user}>
-										<Route exact path="/changePassword">
-											<ChangePasswordPage setUser={setUser} user={user} />
-										</Route>
-										<Route exact path="/profile">
-											<Profile setUser={setUser} user={user} />
-										</Route>
-										<AdminArea user={user}>
-											<Route exact path="/admin">
-												<AdminPage setUser={setUser} user={user} />
+									<Route path="/user">
+										<UserArea user={user}>
+											<Route exact path="/user/changePassword">
```

Comment:
Sorry for the dumb question: what is the advantage of introducing a new filter `/user/`?

Reply:
By introducing a new `/user/` path, the <UserArea> component is only rendered if `/user/` is present in the URL. This allows the <UserArea> to redirect any non users to the front page. If we would not have a `/user/` path, then the component would always be called and would redirect any non user immediately to the landing page even if they are accessing e.g. the FAQ